packages/node/src/edge-functions/edge-handler.ts

```diff
@@ -180,3 +185,13 @@ export async function createEdgeEventHandler(
     };
   };
 }
+
+// turns "api/some.func.js" into "api/some.func"
+function extractUrlPath(entrypointRelativePath: string) {
+  const parts = entrypointRelativePath.split('.');
+  if (parts.length === 1) {
+    return entrypointRelativePath;
+  }
+  // pop of the last element
+  return parts.slice(0, -1).join('.');
```

Comment:
The comment is probably not needed if you use pop 🙂 

```suggestion
  parts.pop();
  return parts.join('.');
```

Reply:
ah, yeah. I was trying to deal with `pop` not returning the new array, but after I moved this to a function, splitting it out on a new line is more reasonable.